prov/rxd/src/rxd_msg.c

```diff
@@ -227,12 +230,15 @@ ssize_t rxd_ep_generic_recvmsg(struct rxd_ep *rxd_ep, const struct iovec *iov,
 				    unexp_list);
 		goto out;
 	}
-
 	if (!(flags & FI_DISCARD)) {
+		rxd_addr = ofi_idx_lookup(&(rxd_ep_av(rxd_ep)->fi_addr_idx), 
+					   RXD_IDX_OFFSET(addr));
+		if(!rxd_addr)
```

Comment:
nit: Please add a space after 'if' and before the "(".  There are several places where this occurs.  :)